decidim-core/app/models/decidim/organization.rb

```diff
@@ -16,5 +16,7 @@ class Organization < ApplicationRecord
     def homepage_big_url
       homepage_image.big.url
     end
+
+    has_many :features, through: :participatory_processes
```

Comment:
Move this together with the other associations. Although I don't see why we need this, why should we query the organization instead of the process?